lib/memjs/memjs.js

```diff
@@ -401,6 +402,35 @@ Client.prototype.stats = function(callback) {
   }
 }
 
+
+// STATS
+//
+// Fetches memcache stats from each connected server. The callback is invoked 
+// **ONCE PER SERVER** and has the signature:
+//
+//     callback(err, server, stats)
+//
+// _server_ is the `"hostname:port"` of the server, and _stats_ is a
+// dictionary mapping the stat name to the value of the statistic as a string.
+Client.prototype.stats = function(callback) {
+  this.statsWithKey('', callback);
+}
+
+// RESET_STATS
+//
+// Reset the statistics each server is keeping back to zero. This doesn't clear
+// stats such as item count, but temporary stats such as total number of
+// connections over time.T
+//
+// The callback is invoked **ONCE PER SERVER** and has the signature:
+//
+//     callback(err, server)
+//
+// _server_ is the `"hostname:port"` of the server.
+Client.prototype.resetStats = function(callback) {
```

Comment:
I think it makes sense, in the spirit of the library, for this to just be done with `statsWithKey('reset')` instead of having a special function for it.